### ISSUE TITLE
Make website look... better

### ISSUE THREAD

**Issue by ryapric** [opensourcecorp/website#1]
There is currently some incredibly rudimentary CSS powering the style of the OSC website -- barely enough to make it other-than-plain-HTML. The site needs some love in the form of more robust CSS styling.

Conditions:
- ***NO JavaScript***, except (if applicable) where "needed" to allow for dynamic viewing between desktop & mobile. Pure CSS for that is preferred, though.
- Total CSS from changes should not exceed ~2.5kB in size (currently, `static/main.css` is 1.4kB)

**Comment by ghost** [comment]
@ryapric please assign me the task. I can surely do this.

**Comment by DhruvDhari** [comment]
@ryapric can i also contribute ?

**Comment by ryapric** [comment]
Hey folks -- no need for assignment, you just need to open PRs from a fork 🙂

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/112039921/193747471-01b95d74-780d-4e6b-9282-c49d4121bcdf.png)
It's your website??
Actually I am new to github so little bit confused where your actual website is located?

**Comment by ryapric** [comment]
Yep, that's the correct address for the website. The `README` in this repo explains & links to how the website is generated & how it works -- but the website content itself is created from this repo.

**Comment by gedw99** [comment]
https://github.com/adityatelange/hugo-PaperMod is pretty good
